tpm

```diff
@@ -21,8 +21,17 @@ tpm_path_set() {
 	tmux show-environment -g "$DEFAULT_TPM_ENV_VAR_NAME" >/dev/null 2>&1
 }
 
+# Check if configuration file exists at an XDG-compatible location, if so use
+# that directory for TMUX_PLUGIN_MANAGER_PATH. Otherwise use $DEFAULT_TPM_PATH.
 set_default_tpm_path() {
-	tmux set-environment -g "$DEFAULT_TPM_ENV_VAR_NAME" "$DEFAULT_TPM_PATH"
+	local XDG_TMUX="${XDG_CONFIG_HOME:-$HOME/.config}/tmux"
+	local TPM_PATH="$DEFAULT_TPM_PATH"
```

Comment:
Since `TPM_PATH` is a local variable can we downcase it as well?

Reply:
Fixed.